templates/github/.github/workflows/release.yml.j2

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.8"
+          python-version: "{{ python_version }}"
```

Comment:
We only need to be concerned with having the CI run on Python 3.6 - the release scripts should run on either 3.8 or "latest" because they break on 3.6

https://github.com/pulp/pulp_rpm/runs/3147430536?check_suite_focus=true#step:7:82

Reply:
```
diff --git a/templates/github/.github/workflows/release.yml.j2 b/templates/github/.github/workflows/release.yml.j2
index 9e4e16e..4e1616a 100644
--- a/templates/github/.github/workflows/release.yml.j2
+++ b/templates/github/.github/workflows/release.yml.j2
@@ -32,7 +32,7 @@ jobs:
 
       - uses: actions/setup-python@v2
         with:
-          python-version: "{{ python_version }}"
+          python-version: 3.8
 
       - name: Install python dependencies
         run: |
@@ -211,7 +211,7 @@ jobs:
 
       - uses: actions/setup-python@v2
         with:
-          python-version: "{{ python_version }}"
+          python-version: "3.8"
 
       - uses: actions/setup-ruby@v1
         with:

```